src/Assignments/SimpleProblemPage.tsx

```diff
@@ -210,18 +210,18 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
 
     const confirmEndVersion = (actualAttemptsRemaining?: number) => {
         actualAttemptsRemaining = actualAttemptsRemaining ?? attemptsRemaining;
-        let message = '';
+        let message = 'You have successfully completed this exam.';
```

Comment:
I personally prefer not to have the success message as the fallback
If there was an unaccounted error it would say success 🤣

Reply:
next line, `if (actualAttemptsRemaining > 0)` ;P

Reply:
🤣 Sounds good should be fine
I would just do it the opposite way in case things change in the future, I want success to mean everything succeeded rather than nothing failed, for now, this is fine though